packages/babel-core/test/api.js

```diff
@@ -12,7 +12,8 @@ import pluginSyntaxJSX from "@babel/plugin-syntax-jsx";
 import pluginFlowStripTypes from "@babel/plugin-transform-flow-strip-types";
 import { itBabel8, commonJS } from "$repo-utils";
 
-const { __dirname: cwd } = commonJS(import.meta.url);
+const { __dirname } = commonJS(import.meta.url);
+const cwd = __dirname;
```

Comment:
Would supporting `const cwd = commonJS(import.meta.url).__dirname` make the plugin significantly more complex?

Reply:
The current `babelImportMetaUrl` plugin removes `const { __dirname } = commonJS(import.meta.url)` and let `__dirname` references the global variable, so it breaks when we rename the variable in destructuring. At first I replaced `commonJS(import.meta.url)` by `{ __dirname, __filename, require }` but the test throws with `_dirname` (one underscore) not defined, I guess it is a glitch in the replacer as it renames `__dirname` to `_dirname` (incorrectly). I will investigate this issue later.

Since it is very likely that we will get rid of the custom `babelImportMetaUrl` in Babel 8, I workaround this issue and just keep this PR mergeable.